UN-OCHA/rwint9-site
# [RW-814] Use cd-disclosure component

Refs: RW-814

This replaces the `cd-dropdown` by the `cd-disclosure` component introduced in https://github.com/UN-OCHA/common_design/pull/427 for the disaster/country long descriptions.

**Note**: this is blocked until there is a new release of the CD with that component, in which case, we need to run a `composer require unocha/common_design:^9.2` and commit the changes to the composer files.

## Tests

1. Checkout the branch, run `composer install` or recreate the local image, clear the cache
2. Go to `/disaster/ep-2013-000175-cod` and compare the `Show description`'s behavior with the one on prod: /disaster/ep-2013-000175-cod, should be unchanged.
